examples/fluentui-app/yarn.lock

```diff
@@ -0,0 +1,11709 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
+# yarn lockfile v1
+
+
+"@babel/code-frame@7.10.4":
```

Comment:
I tried running `yarn build` in `examples/fluentui-app` and it failed. Then I noticed this `yarn.lock`. 

This repo is set up as a monorepo, so you should only have to run `yarn` in repo root. This `yarn.lock` file shouldn't exist, and users shouldn't have to run `yarn` in subpackages. Please add `examples` to `lerna.json`, do a `git clean -fdx` in repo root, run `yarn` and confirm everything runs in the example as you expect.

Reply:
Oh, sure, will do! However, `git clean -dfx` is not a good suggestion to give to contributors - if someone is not very well versed with git, this will delete changes that were not staged yet effectively undoing their work. It is almost always better to hint on using interactive mode instead `git clean -dix`.

Reply:
I'm not giving this advice to random contributors, I'm giving it to you, and you should be well versed in git.